tests/unit/flow/test_flow.py

```diff
@@ -747,3 +749,71 @@ def test_flow_common_kwargs():
 def test_flow_set_asyncio_switch_post(is_async):
     f = Flow(asyncio=is_async)
     assert inspect.isasyncgenfunction(f.post) == is_async
+
+
```

Comment:
can you mark it as a slow test?
```suggestion
pytest.mark.slow
```